.github/workflows/test.yml

```
@@ -0,0 +1,37 @@
+name: Test
+
+on:
+  push:
+
+jobs:
+  test:
+    name: Node ${{ matrix.node }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node: ['12', '14', '16']
```

Ideally I'd like to test against the oldest version we support (12), the LTS version, and the latest version. The [docs](https://github.com/actions/setup-node#supported-version-syntax) make it sound like maybe they support this nvm syntax?

```suggestion
        node: ['12', 'lts/*', 'node']
```